### PR TITLE
Tags doc release note changes for 3.0

### PR DESCRIPTION
**Pull request by scottmarlow**
Signed-off-by: Scott Marlow <smarlow@redhat.com>

For https://github.com/eclipse-ee4j/jstl-api/issues/132

### CODE REVIEW

File: install/jstl/docs/ReleaseNotes-jstl.html [hunk 1]
```diff
@@ -1,190 +1,190 @@
...
+          Tag Library 3.0 and its requirements.
+          <hr>
+          <p></p>
+        </li>
+        <li>Jakarta EE Standard Tag Library TCK tests run on the following
+          platforms:
+          <ul>
+            <li>TBD and possible remove the following two entries</li>
+            <li>Alpine Linux v3.12</li>
+            <li>CentOS Linux 7</li>
```
Comment (gurunrao) [inline install/jstl/docs/ReleaseNotes-jstl.html:155]: depends on https://github.com/eclipse-ee4j/jakartaee-tck/pull/788/files#r769309556

File: install/jstl/docs/ReleaseNotes-jstl.html [hunk 2]
```diff
@@ -1,190 +1,190 @@
...
+          </ul>
+        </li>
+        <hr> </a>
+      <h2><a name="ReferenceRuntime"></a><a name="install_setup_run">Installing,
+          Setting Up, and Running the A Standard Tag Library for JavaServer
+          Pages<small><sup>TM</sup></small> TCK</a></h2>
+      <p>Refer to the<em> Jakarta EE Standard Tag Library TCK 3.0 User's Guide</em>
+        (<a href="./html-usersguide/title.html" title="Jakarta EE Standard Tag Library TCK 2,0 User's Guide (HTML)">HTML</a>,
```
Comment (gurunrao) [inline install/jstl/docs/ReleaseNotes-jstl.html:163]: do we need to retain 2,0 or change to 3.0 or 3,0?

Comment (scottmarlow) [reply install/jstl/docs/ReleaseNotes-jstl.html:163]: I think it should be `3.0`

File: install/jstl/docs/ReleaseNotes-jstl.html (hunk 1 above)
Comment (scottmarlow) [reply install/jstl/docs/ReleaseNotes-jstl.html:155]: commented on ^ link

**Comment by scottmarlow** [comment]
Just made one more version correction and also switched platforms list to contain `Debian`

**Pull request merged**